<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>807</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Configuration Window</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>471</width>
      <height>211</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Wind speed</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rotor speed (RPM)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Pitch</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Yaw</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Solver</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>AeroDyn (BEM)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OpenFOAM (ALM)</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Setting up the simulation</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>481</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="MidFidelity">
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>471</width>
        <height>421</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>turbinesFoam simulation</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>End time</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Write interval</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_11">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>160</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>off</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>on</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Dynamic stall</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>End effects model</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_10">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>190</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Shen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Glauert</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>liftingLine</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>130</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>100</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Set case folder</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>261</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QFrame" name="frame_6">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>230</y>
         <width>371</width>
         <height>181</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>120</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Send files to HPC</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>351</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Server </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_14">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>40</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>151</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Destination in HPC</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>User name</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Generate case files</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="LowFidelity">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>471</width>
        <height>421</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>260</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Enter save file name below</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>130</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>261</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>200</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Plot Cp</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>261</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Save Data</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Set solver folder</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_2">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>90</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Run AeroDyn</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>60</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Set case folder</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Plot Ct</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>280</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>311</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>AeroDyn simulation</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Default">
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>471</width>
        <height>421</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>361</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>This GUI sets up the aerodynamic simulations.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>311</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Currently, two solvers are supported:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>421</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>1. AeroDyn (using blade element momentum theory)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>421</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>2. OpenFoam (using actuator line model)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>421</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>To start, please select the solver!</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
